README.md

```diff
@@ -44,8 +44,15 @@ While many proteins fold into regular conformations which can be easily analyzed
 
 ## Community Guidelines
 
-------------------------
+ We welcome the members of open-source community to extend the functionalities of TBD, submit feature requests and report bugs.
+ 
+ ### Feature Request:
+ If you would like to suggest a feature or start a discussion on possible extension of TBD, please feel free to <a href="https://github.com/Intrinsically-Disordered/TBD/issues/new">raise an issue</a>.
+ 
+ ### Bug Report:
+ If you would like to report a bug, please follow <a href="https://github.com/Intrinsically-Disordered/TBD/issues/new">this link</a>.
+ 
```

Comment:
So the bug report and feature request are supposed to be the same link?

Reply:
That's how HARDy did it, but I feel like that makes sense